### DISCUSSION TITLE
cost of a the professional software

### DISCUSSION THREAD

**Discussion by LovesPictures** [mozilla/hubs#3326] [Spoke Questions] [answered]
Hello 

I am trying to figure out how to cost a professional account is; I plan to use the spoke/hubs with students, I need a definitive annual figure. I will also need to know what happens to hubs spaces when this service lapses - do we lose our work or does it revert to a general account?

thanks

**Comment by pattersonbl2** [comment, marked as answer]
Currently, We don't have a professional account option, you can find  information for our self hosted hubs cloud option here: https://hubs.mozilla.com/cloud
